### ISSUE TITLE
http.METHODS is missing PATCH and others

### ISSUE THREAD

**Issue by tjmehta** [jhiesey/stream-http#10]
Hello @jhiesey,

I noticed after upgrading to `browserify@11.0.1`. Our team specifically noticed `require('http').METHODS` was missing `'PATCH'`. Is there any reason why this is hardcoded to https://github.com/jhiesey/stream-http/blob/master/index.js#L51? If not can you please add `'PATCH'` and others to this list maybe you can use https://www.npmjs.com/package/methods?

**Comment by feross** [comment]
Hm, that's a good point. In node 0.12, I get:

```
> http.METHODS
[ 'CHECKOUT',
  'CONNECT',
  'COPY',
  'DELETE',
  'GET',
  'HEAD',
  'LOCK',
  'M-SEARCH',
  'MERGE',
  'MKACTIVITY',
  'MKCOL',
  'MOVE',
  'NOTIFY',
  'OPTIONS',
  'PATCH',
  'POST',
  'PROPFIND',
  'PROPPATCH',
  'PURGE',
  'PUT',
  'REPORT',
  'SEARCH',
  'SUBSCRIBE',
  'TRACE',
  'UNLOCK',
  'UNSUBSCRIBE' ]
```

**Comment by jhiesey** [comment]
Released as v1.4.0

**Comment by tjmehta** [comment]
Great, thanks :+1:
